README.rst

```diff
@@ -1,54 +0,0 @@
-===============
```

Comment:
While replacing `README.rst` with `README.md` is fine, I believe that `README.rst` was chosen originally for its compatibility with https://pypi.org/ - @sloretz, can you confirm?